### ISSUE TITLE
Custom Fonts Do No Show in the Default Theme Builder Font Picker

### ISSUE THREAD

**Issue by joshgk00** [vanjarosoftware/Vanjaro.Platform#406]
I added a custom font but the font doesn't show up in Default Theme Builder to use for the Breadcrumb and other site elements. It does show up in the Typography settings for Text Blocks, etc. 

What I would expect to happen is that after a custom font is added, that it would be an available option to the Default Theme Builder to use site-wide

Steps to Reproduce:

- Go to SEO & Site Search and link the style sheets to a custom Google Font, **Open Sans**
  `<link rel="preconnect" href="https://fonts.gstatic.com">`
  `<link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap" rel="stylesheet">`
  
![image](https://user-images.githubusercontent.com/13650025/101813768-2ce71780-3aeb-11eb-83e3-f5bd6f7640b2.png)

- Go to Custom Theme Builder and add the Open Sans Font
  Font Name = Open Sans
  Font Family = 'Open Sans', sans-serif
  ![image](https://user-images.githubusercontent.com/13650025/101813953-67e94b00-3aeb-11eb-98b0-5227e7dedc6c.png)

- Save the theme
- Clear the Client Cache
- Clear the Server Cache
- Restart the Application
- Go back to the Default Theme Builder
   ![image](https://user-images.githubusercontent.com/13650025/101814263-e514c000-3aeb-11eb-8eeb-100b194ec9e4.png)

- Expand Breadcrumb

At this point, I would expect to see the new font listed in the drop down so that it can be applied to these global settings.

**Comment by Tychodewaard** [comment]
The custom font is meant to be setup a little bit differently. https://www.youtube.com/watch?v=lIySOo8-Z3k
Mind you, that if you do so, it might cause the site to crash as stated in #346 

Use the DNN persona bar to the import as a temp fix:
![image](https://user-images.githubusercontent.com/9091997/101815223-8832fd00-3b1f-11eb-88d2-111d998281b3.png)

**Comment by joshgk00** [comment]
I experienced the same issue with the site crashing trying to setup the Custom font the way that it was intended. 

For some reason when I tried to add the font using the work-around you suggested, that also messed up the front end of my site. 

The only I've had success was importing the stylesheets and then adding classes that reference the font family.

Oddly enough, when I setup the custom font the way I documented in this issue, it works for Text Blocks when you go to the Styling section. 
![image](https://user-images.githubusercontent.com/13650025/101816180-b2b89200-3aee-11eb-8a32-771539bcb1a4.png)
